h2o-persist-http/src/main/java/water/persist/PersistHTTP.java

```diff
@@ -41,7 +42,7 @@
     final long offset = (k._kb[0] == Key.CHK) ? FileVec.chunkOffset(k) : 0L;
 
     URI source = decodeKey(k);
-    HttpGet req = new HttpGet(source);
+    HttpRequestBase req = createReq(source, false);
     req.setHeader(HttpHeaders.RANGE, "bytes=" + offset + "-" + (offset+v._max-1));
     req.setHeader(HttpHeaders.ACCEPT_ENCODING, "identity");
```

Comment:
We can now remove this line as it's already specified inside of the `createReq` method

Reply:
Good catch, Kuba! Thanks for spotting that!